src/Components/ImageOrVideo.tsx

```diff
@@ -26,8 +26,22 @@ export const ImageOrVideo = connect(function ImageOrVideo<T extends string>({
   }>
 }) {
   const [isPaused, setIsPaused] = useState(false)
+  const [shouldAutoplay, setShouldAutoplay] = useState(true)
   const videoRef = useRef<HTMLVideoElement>(null)
 
+  useEffect(() => {
+    const mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)')
+    setShouldAutoplay(!mediaQuery.matches)
+    if (mediaQuery.matches) setIsPaused(true)
```

Comment:
* [ ] We should definitely not put the play button on top. For reduced motion it may cover elements the user is supposed to interact with by default.

Reply:
> Adding a second video tag feels like a hack to me

The root cause is that we're using a setting that was introduced for CSS to control a DOM element. Hiding/showing DOM elements is one of the few things CSS can do to "manipulate" the DOM.
With this in mind, JS media queries also feel like a hack.
Currently our hack renders a non-autoplaying video as autoplaying in prerendered HTML (and on navigating, for a brief moment), because of how we initialize the state and the use of `useEffect` instead of `useLayoutEffect`. If we stick with the JS version we should at least

* [ ] Render non-playing initially